packages/components/src/input-control/test/index.js

```diff
@@ -42,48 +48,52 @@ describe( 'InputControl', () => {
 	} );
 
 	describe( 'Ensurance of focus for number inputs', () => {
-		it( 'should focus its input on mousedown events', () => {
+		it( 'should focus its input on mousedown events', async () => {
+			const user = setupUser();
 			const spy = jest.fn();
 			render( <InputControl type="number" onFocus={ spy } /> );
+			const target = getInput();
 
-			const input = getInput();
-			fireEvent.mouseDown( input );
+			// Hovers the input and presses (without releasing) primary button.
+			await user.pointer( [
+				{ target },
+				{ keys: '[MouseLeft]', target },
+			] );
 
 			expect( spy ).toHaveBeenCalledTimes( 1 );
 		} );
 	} );
 
 	describe( 'Value', () => {
-		it( 'should update value onChange', () => {
+		it( 'should update value onChange', async () => {
+			const user = setupUser();
 			const spy = jest.fn();
 			render( <InputControl value="Hello" onChange={ spy } /> );
-
 			const input = getInput();
-			input.focus();
-			fireEvent.change( input, { target: { value: 'There' } } );
 
-			expect( input.value ).toBe( 'There' );
-			expect( spy ).toHaveBeenCalledTimes( 1 );
+			await user.type( input, ' there' );
+
+			expect( input ).toHaveValue( 'Hello there' );
+			expect( spy ).toHaveBeenCalledTimes( 6 );
 		} );
 
-		it( 'should work as a controlled component', () => {
+		it( 'should work as a controlled component', async () => {
+			const user = setupUser();
 			const spy = jest.fn();
 			const { rerender } = render(
 				<InputControl value="one" onChange={ spy } />
 			);
-
 			const input = getInput();
 
-			input.focus();
-			fireEvent.change( input, { target: { value: 'two' } } );
+			await user.type( input, '2' );
 
 			// Ensuring <InputControl /> is controlled.
-			fireEvent.blur( input );
+			await user.click( document.body );
```

Comment:
> Also on the general idea of what is the most clear way to blur elements, I don't really have an opinion. I'd be glad to use user.tab() instead but so far it seems like our other tests have been going with user.click( document.body ).

I've been using the same technique on `UnitControl` ([example](https://github.com/WordPress/gutenberg/blob/ac07469123f4124844852bdd21b74e62448e3726/packages/components/src/unit-control/test/index.tsx#L295-L296)), and just added a comment to clarify my intentions.

We could also look into adding some "blur" utility function to our testing utils, in case that makes it more explicit/

Reply:
Agreed with all points above:

- up for switching the unit test to the one proposed by @stokesman in https://github.com/WordPress/gutenberg/pull/41421#discussion_r888270852
- I say, let's keep writing explicit clicks on `document.body` for now, and make sure that they are well commented out. We can keep an eye on this pattern and decide to refactor to a utility later on if we feel it's necessary